README.md

```diff
@@ -13,6 +13,7 @@ Handling of large bulk data into database tables.
 - Full load - destination table is truncated before load
 - Incremental load - data are merged
 - Primary key dedup for all engines
+- Convert empty strings to `NULL` (using `nullify` option)
```

Comment:
asi bych to fakt radši pojmenoval `convertEmptyStringToNull` nebo `convertEmptyValuesToNull`.